devices/Sim7080/packages.config

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
for the category file, please use category "wireless", I think it fits there properly

Reply:
@Ellerbach I suggest that we add a category to have a clearer distinction on the device. Wireless is obviously correct, but, at  a glance, one  "can't" understand imediatly if it's local or long range, Wi-Fi, LoRa, NBIoT, etc.

Reply:
I will add wireless for now, but maybe  'radio communication' covers it?